wallet/createtx.go

```diff
@@ -234,6 +241,7 @@ func createTx(eligible []wtxmgr.Credit,
 			if err != nil {
 				return nil, err
 			}
+
```

Comment:
Everything looks good now except this extra newline added.  Can you rebase+squash?